Update 'Using a specific shell' (#7824)
github/docs: content/actions/reference/workflow-syntax-for-github-actions.md

### Using a specific shell

> You can override the default shell settings in the runner's operating system using the `shell` keyword. You can use built-in `shell` keywords, or you can define a custom set of shell options. The shell command run internally will execute a temporary file, containing the commands you specify in each `run` keyword.

Reviewer: ```suggestion
You can override the default shell settings in the runner's operating system using the `shell` keyword. You can use built-in `shell` keywords, or you can define a custom set of shell options. The shell command that is run internally executes a temporary file that contains the commands specifed in the `run` keyword.
```
